File: src/main/resources/output/IFC4X3_csharp/Entities/IfcExternalReference/IfcLibraryReference/IfcLibraryReference.cs
public class IfcLibraryReference : IfcExternalReference
{
    public IfcText Description { get; set; }
    public IfcLanguageId Language { get; set; }
    public IfcLibraryInformation ReferencedLibrary { get; set; }

    // === INVERSE CLAUSES ===
    // LibraryRefForObjects : SET [0:?] OF IfcRelAssociatesLibrary FOR RelatingLibrary
}

File: src/main/resources/output/IFC4X3_vbnet/Entities/IfcExternalReference/IfcLibraryReference/IfcLibraryReference.vb
Public Class IfcLibraryReference Inherits IfcExternalReference
    Public Description As IfcText
    Public Language As IfcLanguageId
    Public ReferencedLibrary As IfcLibraryInformation

    ' === INVERSE CLAUSES ===
    ' LibraryRefForObjects : SET [0:?] OF IfcRelAssociatesLibrary FOR RelatingLibrary
End Class
